test/sourcemap-test.coffee

```diff
@@ -27,6 +27,6 @@ describe "Parser sourcemap", ->
 
   # Parser Sourcemap should conform to recent source map serialization JSON media type
   it '`sourcemap` field conforms to `vnd.apiblueprint.sourcemap+json; version=2.1`', ->
-    console.log JSON.stringify sourcemap_parsed, undefined, 2
+    # console.log JSON.stringify sourcemap_parsed, undefined, 2
```

Comment:
This was not commented in the original code by accident. When running tests, this prints a lot of lines thus distracting me from the actual test report. So, I commented it out.